packages/console/src/pages/ConnectorDetails/ConnectorContent/index.tsx

```diff
@@ -112,6 +112,13 @@ function ConnectorContent({ isDeleted, connectorData, onConnectorUpdated }: Prop
     })
   );
 
+  const updateUsage = useCallback(() => {
+    if (connectorData.usage === undefined) {
+      return;
+    }
+    onConnectorUpdated({ ...connectorData, usage: connectorData.usage + 1 });
```

Comment:
how about fetching from remote?

Reply:
Can do this later, see details in [LOG-6529](https://linear.app/silverhand/issue/LOG-6529).

Reply:
Can do this later, see details in [LOG-6529](https://linear.app/silverhand/issue/LOG-6529).